frontend/public/locales/en/status.json

```diff
@@ -48,5 +48,11 @@
     "hcaptcha": "https://www.hcaptcha.com/terms",
     "microsoft-data-privacy-policy": "https://www.microsoft.com/en-ca/trust-center/privacy",
     "microsoft-service-agreement": "https://www.microsoft.com/en-ca/servicesagreement"
+  },
+  "form": {
+    "application-code-label": "Application Code or Client Number (required)",
```

Comment:
`(required)` should be removed since the InputField have the asterix and hidden `(required)` for a11y.